app/connectors/TrustClaim.scala

```diff
@@ -23,41 +23,41 @@ import play.api.libs.json.{Json, OFormat}
 
 import scala.language.implicitConversions
 
-case class TrustClaim(utr:String, managedByAgent: Boolean, trustLocked:Boolean)
+case class TrustClaim(identifier:String, managedByAgent: Boolean, trustLocked:Boolean)
```

Comment:
@BriWak Can you make sure the response from trusts-store aligns to this and also returns the field identifier in it's response?